tox.ini

```diff
@@ -14,7 +15,9 @@ deps =
     https://github.com/pallets/markupsafe/archive/master.tar.gz
     https://github.com/pallets/jinja/archive/master.tar.gz
     https://github.com/pallets/itsdangerous/archive/master.tar.gz
-    https://github.com/pallets/click/archive/master.tar.gz
+
+    click8: https://github.com/pallets/click/archive/master.tar.gz
```

Comment:
Remove the click8 tag so that `tox -e py` still works.

Reply:
I didn't know `!click7` works (I need to specify something or pip will choke on the dependency conflict) since the documentation on those markers is VERY poor... but apparently it works just fine so I'm using that now.